gnosis/safe-react
# Feature/2282 use config safe app api

## What it solves

Closes #2282 

## How this PR fixes it

Makes an axios request to Safe Apps API to get the list of default Safe Apps

## How to test it

- Open the Apps section
- Add a custom safe app
- If the endpoint fails it should show the local saved safe apps

## Any extra information
